src/google/adk/tools/toolbox_tool.py

```diff
@@ -23,7 +23,7 @@ class ToolboxTool:
 
   Example:
   ```python
-  toolbox = ToolboxTool("http://127.0.0.1:8080")
+  toolbox = ToolboxTool("http://127.0.0.1:5000")
```

Comment:
toolbox server default port is `5000` so updating it to match

https://github.com/googleapis/genai-toolbox/blob/0a7d3ff06b88051c752b6d53bc964ed6e6be400e/cmd/root.go#L106